lib/services/nsStarterKitsNpmService.ts

```diff
@@ -0,0 +1,82 @@
+import util from "../shared/util";
+
+const defaultHeaders = {
+    "user-agent": "nativescript-starter-kits"
+};
+
+export class NsStarterKitsNpmService implements INsStarterKitsNpmService {
+    installPageTemplateFromNpm(pageName: string, flavor: string, templatesDirectory: string): Promise<any> {
```

Comment:
Do we need ...FromNpm(...) in the method name?
As it is defined in NpmService class, I think installPageTemplate(...) is enough.